### DISCUSSION TITLE
How to Change a user's e-mail address?

### DISCUSSION THREAD

**Discussion by haroldtheb** [docker-mailserver/docker-mailserver#4134] [Q&A] [answered]
Is there a simple way to change a user's email address?  For example, I need to move a user from email1@example.com to email2@example.com.  Thanks in advance.

**Comment by casperklein** [comment, marked as answer]
Given, that you use our default [`compose.yaml`](https://github.com/docker-mailserver/docker-mailserver/blob/32c0a346fcec975ba133443d98e3a1e7c0d78714/compose.yaml) file:

1. Stop DMS
2. Search and replace `email1@example.com` with `email2@example.com` in `./docker-data/dms/config/postfix-accounts.cf`
3. `mv ./docker-data/dms/mail-data/example.com/email1 ./docker-data/dms/mail-data/example.com/email2`
4. Start DMS (`docker compose up --force-recreate`).

I've **not** tested the steps above, but they should work. In any case, backup your data before you make any changes.

**Reply by haroldtheb** [reply]
Thank you very much! This worked flawlessly
